src/playground/Pages/Tabs/index.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import AreaChartStory from "@/components/Charts/AreaChart/index.stories"
+import { AreaChartInsight } from "@/components/Insights/Charts/AreaChartInsight"
+import { AutoGrid } from "@/components/Layout/AutoGrid"
+import type { Meta, StoryObj } from "@storybook/react"
+import { Tabs } from "."
+
+const meta = {
+  component: Tabs,
+  tags: ["autodocs"],
+} satisfies Meta<typeof Tabs>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+const Overview = () => (
+  <div className="grid grid-cols-[2fr_1fr] divide-x divide-y-0 divide-dashed divide-muted">
+    <AutoGrid tileSize="md" className="col-span-2 pl-10 pr-8 pt-6">
```

Comment:
We just recently added a `Dashboard` component meant for this use case! It just uses `AutoGrid` but I think it's better for scalability.